addons/escoria-core/game/esc_autoload.gd

```diff
@@ -149,38 +152,38 @@ func set_game_paused(p_paused: bool):
 		scene_tree.paused = p_paused
 
 
-# Apply the loaded settings
+## Apply the loaded settings
```

Comment:
Ditto here. Let's remove the commented blocks of code.

Reply:
Done